packages/rum-core/src/domain/rumEventsCollection/view/viewCollection.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('viewCollection', () => {
         resourceCount: 10,
         userActionCount: 10,
       },
-      id: 'xxx',
+      id: 'aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee',
```

Comment:
what is the purpose of this change?

Reply:
The view id has to respect a certain format compliant with rum-events-format

Reply:
🤦‍♂️ indeed